test/number/n_GF-test.jl

```diff
@@ -5,8 +5,9 @@
    @test elem_type(N_GField) == n_GF
    @test parent_type(n_GF) == N_GField
    @test base_ring(R) == Union{}
+   @test isa(x, n_GF)
 
-   typeof(R) <: Nemo.Field
+   @test typeof(R) <: Nemo.Field
```

Comment:
More "idiomatic" might be `@test R isa Nemo.Field`, but not worth bothering.

Reply:
good point; I didn't even think about what the code does, just that those tests were not actually doing anything...

Reply:
I've adjusted this in a bunch of tests now